src/Sources/Abstractions/src/LangChain.Sources.Abstractions.csproj

```diff
@@ -14,6 +14,11 @@
         <Using Remove="System.Net.Http" />
     </ItemGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Microsoft.AspNetCore.StaticFiles" />
```

Comment:
This package is deprecated, also I don't think it should be in Abstractions package

Reply:
resolved